setup.py

```diff
@@ -17,8 +17,24 @@
       dependencies.""",
       long_description=long_description,
       # Get strings from http://www.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=[],
-      keywords=[],
+      classifiers=[
+          'Development Status :: 5 - Production/Stable',
+          'Environment :: Console',
+          'Intended Audience :: Developers',
+          'License :: OSI Approved :: Zope Public License',
```

Comment:
Does it make sense to keep this license with the complete rewrite that I did? There are no longer any parts left that are coming from the old code. I would be happier to change the license to (L)GPLv3 or similar, but I'm not a lawyer, so I have no clue on these things 😅

Reply:
Getting rid of the ZPL: good idea. (L)GPL or BSD, all three are fine.

Most common nowadays: BSD, which means that everyone can copy your code around and can even use it in commercial products and change the license and so. GPL is more strict in that it has to stay open source and that software that uses it also has to be GPL. LGPL allows you to use it in non-GPL code.

Normally I'd personally use LGPL, but in the case of this tool I'd say BSD. It is makes it a bit easier to (re-)use it in other tools, which is probably what we want in the end.